assets/panel/debugger.properties

```diff
@@ -601,11 +601,11 @@ sourceFooter.codeCoverage=Code coverage
 
 # LOCALIZATION NOTE (sourceFooter.currentCursorPosition): Text associated
 # with the current cursor line and column
-sourceFooter.currentCursorPosition=(%S, %S)
+sourceFooter.currentCursorPosition.label=(%1$S, %2$S)
```

Comment:
No need to update the ID for this one, it's a safe change (localizers can still change it, they're equivalent)